ops/testing.py

```diff
@@ -799,6 +799,9 @@ def add_relation(self, relation_name: str, remote_app: str, *,
 
         This function creates a relation with an application and triggers a
         :class:`RelationCreatedEvent <ops.RelationCreatedEvent>`.
+        It also creates a default network binding on this endpoint. If you want
+        to associate a custom network to this binding, provide one using
+        :meth:`add_network` before calling this function.
```

Comment:
Maybe also a doc comment that these networks are not removed on `remove_relation`?

Reply:
Should we not have `remote_relation` reverse this? WWJD (What Would Juju Do)?

Reply:
huh, very good point.

Reply:
I think that is a complicated choice because we'd have to distinguish between manually and automatically added networks, I guess it'd be confusing if you `.add_network`, then `.add_relation; .remove_relation` and then the network is gone. It does feel cleaner I suppose, but might be surprising.